src/utils/database.js

```diff
@@ -0,0 +1,14 @@
+import { Sequelize } from "sequelize";
+
+const DB_PATH = process.env.DB_PATH;
+const DEV_DB_PATH = 'sqlite::memory:';
+
+export const sequelize = new Sequelize(DEV_DB_PATH);
```

Comment:
This will always connect to `DEV_DB_PATH`? looks like `DB_PATH` is never used

Reply:
yes missed this, will add a env variable of type "prod" || "dev" and will connect accordingly

Reply:
fixed